lib/game.rb

```diff
@@ -1,23 +1,40 @@
 require_relative "./presenter"
 require_relative "./displayer"
+require_relative "./board"
+require_relative "./player"
 class Game
-  attr_reader :display
-  def initialize(display, presenter)
+  attr_reader :display, :board
+  def initialize(display, presenter, board)
     @display = display
     @presenter = presenter
+    @board = board
+    @player_1 = Player.new("X")
+    @player_2 = Player.new("O")
+    @game_over = false
```

Comment:
Why do we need two players? Where do we use @game_over?

Reply:
This is where I started getting ahead of myself. I commented this out for now.